internal/store/containers.go

```diff
@@ -1,170 +0,0 @@
-// Copyright 2020 MongoDB Inc
```

Comment:
Container store have been wrongly pointing to peering connections store. 
Since both methods are available under the same package i have moved those methods for clarity